core/string/grapheme_clusters_spec.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'shared/chars'
+require_relative 'shared/grapheme_clusters'
+
+ruby_version_is "2.5" do
+  describe "String#grapheme_clusters" do
```

Comment:
Could you add a spec that this method returns an Array if called without a block?

Reply:
Sure thing!